src/components/__tests__/Menu.js

```diff
@@ -293,6 +294,27 @@ describe('default render', () => {
 
     expect(toggleShowMore).toHaveBeenCalledTimes(1);
   });
+
+  it('calls the Panel mixin with `canRefine`', () => {
```

Comment:
looks like we're not actually testing the mixin, but that indeed seems hard to attain

Reply:
actually, we are testing it enough in panel so it's ok

Reply:
Yep indeed I'm only testing the function mapping function here.